### ISSUE TITLE
NTR: Liver biopsy: low glucose-6-phosphate translocase activity (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9090]
New term request Liver biopsy: low glucose-6-phosphate translocase activity (segal_060420141636): T1 transport protein; Transporter; hepatic biopsy  UMLS: 1856362

**Comment by pnrobinson** [comment]
@MickeySegal
I am wondering if we should not put liver in the label since
[Type I GSD](https://www.sciencedirect.com/topics/medicine-and-dentistry/glycogen-storage-disease-type-i) is caused by the absence or deficiency ofglucose-6-phosphatase activity in the liver, kidney, and [intestinal mucosa](https://www.sciencedirect.com/topics/medicine-and-dentistry/intestine-mucosa).
and liver is just the easiest place to biopsy

**Comment by MickeySegal** [comment]
That is fine, but we include this in our liver biopsy bundle so we mention the liver.  You don't need to mention the liver.

**Comment by pnrobinson** [comment]
Looking again, it seems liver is the most common biopsy location, so

Adding new term: 
- ID: HP:6000201
- label: Reduced hepatic glucose-6-phosphate translocase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: PMID:9428641
- def: Activity of glucose-6-phosphate translocase in liver below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9090
- synonyms:
